libs/structs/structpath/path.go

```diff
@@ -73,16 +77,36 @@ func (p *PathNode) KeyValue() (key, value string, ok bool) {
 	return "", "", false
 }
 
-func (p *PathNode) IsStringKey() bool {
-	return p != nil && p.index == tagStringKey
+func (p *PathNode) IsDotString() bool {
+	return p != nil && p.index == tagDotString
+}
+
+func (p *PathNode) DotString() (string, bool) {
```

Comment:
Are these methods used anywhere?

Reply:
Would be nice to include a test to confirm they work.